llvm/lib/ObjCopy/ELF/ELFObject.cpp

```diff
@@ -2636,30 +2636,30 @@ template <class ELFT> Error ELFWriter<ELFT>::finalize() {
 }
 
 Error BinaryWriter::write() {
-  SmallVector<const SectionBase *, 30> LoadableSections;
+  SmallVector<const SectionBase *, 30> BitsSections;
```

Comment:
Without knowing the code around it, I wouldn't know what `BitsSections` means. How about `SectionsToWrite`?

Reply:
Done.